index.html

```diff
@@ -14,6 +14,7 @@
             <li><a href="#">Blog</a></li>
             <li><a href="#">Contact</a></li>
         </ul>
+        <img src="./image/image.jpeg" alt="background-image">
       </nav>
     </div>
   </body>
```

Comment:
You did a great job but in case of next time, ensure you add your CSS  so I know what that image is doing. However, add your CSS rule for the image.

<img src="https://media2.giphy.com/media/qN8mwUPAt7gByge3W1/giphy.gif"/>